specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json

```diff
@@ -0,0 +1,1502 @@
+                }
+            }
+        },
+        "ServiceProperties": {
+            "description": "Details about Azure service available for support.",
+            "type": "object",
+            "properties": {
+                "displayName": {
+                    "description": "Localized name of Azure service.",
+                    "type": "string"
+                }
+            }
+        },
+        "ProblemClassificationsListResult": {
+            "description": "Object that represents a collection of ProblemClassification resources.",
+            "type": "object",
+            "properties": {
+                "value": {
+                    "description": "List of ProblemClassification resources.",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ProblemClassification"
+                    }
+                }
+            }
+        },
+        "ProblemClassification": {
+            "description": "Object that represents a ProblemClassification resource.",
+            "type": "object",
+            "properties": {
+                "id": {
+                    "description": "Id of the resource.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "name": {
+                    "description": "Name of the resource.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "type": {
+                    "description": "Type of the resource 'Microsoft.Support/problemClassification'.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/ProblemClassificationProperties",
+                    "description": "Properties of the resource."
+                }
+            }
+        },
+        "ProblemClassificationProperties": {
+            "description": "Details about a problem classification available for an Azure service.",
+            "type": "object",
+            "properties": {
+                "displayName": {
+                    "description": "Localized name of problem classification.",
+                    "type": "string"
+                }
+            }
+        },
+        "CheckNameAvailabilityInput": {
+            "description": "Input of CheckNameAvailability API.",
+            "type": "object",
+            "required": [
+                "name",
+                "type"
+            ],
+            "properties": {
+                "name": {
+                    "description": "The resource name to validate.",
+                    "type": "string"
+                },
+                "type": {
+                    "type": "string",
+                    "enum": [ "Microsoft.Support/supportTickets" ],
```

Comment:
Are there plans to add new resource types to this list in the future?

Reply:
Added one more "Microsoft.Support/communications" to this list. No plans to add more in the near future.